pkg/apis/discovery/types.go

```diff
@@ -34,8 +34,8 @@ type EndpointSlice struct {
 	// addressType specifies the type of address carried by this EndpointSlice.
 	// All addresses in this slice must be the same type. The following address
 	// types are currently supported:
-	// * IP:   Represents an IP Address. This can include both IPv4 and IPv6
-	//         addresses.
+	// * IPv4: Represents an IPv4 Address.
+	// * IPv6: Represents an IPv6 Address.
 	// * FQDN: Represents a Fully Qualified Domain Name.
 	// +optional
 	AddressType *AddressType
```

Comment:
If it has no default value but can't be unspecified, it's not optional.  If it's not optional, it doesn't need to be a pointer (since "" is not valid)

Reply:
Optional tag removed, will work on moving this away from being a pointer.